lib/node/nodes/trade-area/trade-area-query-builder.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
I was thinking more about a generic method to load any node/*.sql file as a `dot.template`.